MANUAL.txt

```diff
@@ -259,7 +259,7 @@ header when requesting a document from a URL:
     - `man` ([roff man])
     - `muse` ([Muse])
     - `native` (native Haskell)
-    - `odt` ([ODT])
+    - `odt` ([OpenOffice text document])
```

Comment:
This will be a broken link, because the reference is not defined.
It would have to be `[OpenOffice text document][ODT]` as in the other case.

Reply:
Please fix it your way. All I know is how it looks on the man page. Thanks.